src/main/kotlin/dsm/wemeet/domain/friend/service/impl/CheckFriendServiceImpl.kt

```
@@ -0,0 +1,12 @@
+package dsm.wemeet.domain.friend.service.impl
+
+import dsm.wemeet.domain.friend.repository.FriendJpaRepository
+import dsm.wemeet.domain.friend.service.CheckFriendService
+import org.springframework.stereotype.Service
+
+@Service
+class CheckFriendServiceImpl(
+    private val friendJpaRepository: FriendJpaRepository
+) : CheckFriendService {
+
+}
```

detekt (1.23.7)

[warning] The class or object CheckFriendServiceImpl is empty.

(detekt.empty-blocks.EmptyClassBlock)